cinema/serializers.py

```diff
@@ -59,6 +61,7 @@
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `tickets_available` field is defined as a read-only `IntegerField`, but it is not clear how this field is populated or calculated. Ensure that there is a method or property in the `MovieSession` model that provides the correct value for `tickets_available`.